Frends.SFTP.ReadFile/Frends.SFTP.ReadFile.Tests/ServerFingerprintTests.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System;
+using NUnit.Framework;
+using Renci.SshNet.Common;
+using Frends.SFTP.ReadFile.Enums;
+
+namespace Frends.SFTP.ReadFile.Tests;
+
+[TestFixture]
+public class ServerFingerprintTests : ReadFileTestBase
+{
+    internal static string _MD5;
+    internal static string _Sha256Hex;
+    internal static string _Sha256Hash;
```

Comment:
All of these properties can be `private` instead of `internal`, since they are only accessed inside this class.